compiler-rt/lib/profile/InstrProfilingUtil.h

```diff
@@ -31,6 +31,25 @@ int lprofUnlockFileHandle(FILE *F);
  * lock for exclusive access. The caller will block
  * if the lock is already held by another process. */
 FILE *lprofOpenFileEx(const char *Filename);
+
+enum MemoryStatus {
+  MM_INVALID = 0x1, // Addr is not a valid address
+  MM_MMAP = 0x2,    // Addr was mmap'ed
+  MM_MALLOC = 0x4   // Addr was malloc'ed
```

Comment:
What does `MM` stand for? Given the `enum` is called `MemoryStatus`, I'd expect these values to use prefix `MS_` as is the case elsewhere in LLVM.

Also, is there any reason why the values are defined as bit flags? As far as I can tell, we never want these to be combined (i.e. no `MM_MMAP | MM_MALLOC`) so the values can use natural ordering.